default/scripting/buildings/SPACE_ELEVATOR.focs.txt

```diff
@@ -12,7 +12,7 @@ BuildingType
     ]
     EnqueueLocation = [[ENQUEUE_BUILD_ONE_PER_PLANET]]
     effectsgroups = [
-        // check GasGiant first, so not another match
+        // gas giant is first in the stacking group in order not to trigger another effect by planet size
```

Comment:
GasGiant is also a planet size.

Reply:
I couldnt figure out where it does the equality check using the enum. is it already specified in [ConditionParser6.cpp](https://github.com/freeorion/freeorion/blob/57425eb8a95d672b90976a46539b62ebbaf234ee/parse/ConditionParser6.cpp#L112)?

Anyway so should I remove the comment and the stackinggroup i introduced?

Reply:
I don't understand the first question.

I don't know what the point of the stacking group is, so I don't know.

Reply:
I thought that the size conditions would check for the numeric size. Now the GG numeric size is 4, same as large, so I thought both the effect for large would kick in for a GG. In order to prevent it i introduced the priority/stackinggroup.